docs-v2/integrations/all/unipile.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Unipile
+sidebarTitle: Unipile
+---
+
+API configuration: [`unipile`](https://nango.dev/providers.yaml)
+
+## Features
+
+| Features | Status |
+| - | - |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Auth (OAuth)](/integrate/guides/authorize-an-api) | 🚫 |
+| [Sync data](/integrate/guides/sync-data-from-an-api) | 🚫 |
```

Comment:
can you change that to be the same format of all the others (all checked except webhooks)

Reply:
I updated the list